src/modules/rover_ackermann/module.yaml

```diff
@@ -0,0 +1,175 @@
+module_name: Rover Ackermann
+
+parameters:
+  - group: Rover Ackermann
+    definitions:
+      RA_WHEEL_BASE:
+        description:
+          short: Wheel base
+          long: Distance from the front to the rear axle
+        type: float
+        unit: m
+        min: 0.001
+        max: 100
+        increment: 0.001
+        decimal: 3
+        default: 0.5
+
+      RA_MAX_STR_ANG:
+        description:
+          short: Maximum steering angle
+          long: The maximum angle that the rover can steer
+        type: float
+        unit: rad
+        min: 0.1
+        max: 3.14
```

Comment:
Is an angle >90° geometrically even possible to handle by the guidance controller?

Reply:
Meant to fix that after the last review but thought PI is 90 degrees :')